albumentations/augmentations/domain_adaptation.py

```diff
@@ -200,3 +220,85 @@ def get_transform_init_args_names(self):
 
     def _to_dict(self):
         raise NotImplementedError("FDA can not be serialized.")
+
+
+class PixelDistributionAdaptation(ImageOnlyTransform):
+    """
+    Another naive and quick pixel-level domain adaptation. It fits a simple transform (such as PCA, StandardScaler
+    or MinMaxScaler) on both original and reference image, transforms original image with transform trained on this
+    image and then performs inverse transformation using transform fitted on reference image.
+
+    Args:
+        reference_images (List[str] or List(np.ndarray)): List of file paths for reference images
+            or list of reference images.
+        blend_ratio (float, float): Tuple of min and max blend ratio. Matched image will be blended with original
+            with random blend factor for increased diversity of generated images.
+        read_fn (Callable): Used-defined function to read image. Function should get image path and return numpy
```

Comment:
I think would be great to describe a function example in docstring, or at least the function arguments.

Reply:
Extended the comment a bit. However, please keep in mind that I just took this part from the existing HistogramMatching augmentation.